TypeCobol.LanguageServer/VsCodeProtocol/Initialize Method/ServerCapabilities.cs

```diff
@@ -85,5 +85,10 @@ public class ServerCapabilities
         /// The server provides rename support.
         /// </summary>
         public bool renameProvider { get; set; }
+
+        /// <summary>
+        /// Experimental value.
+        /// </summary>
+        public object experimental;
```

Comment:
Why using the word `experimental` ?

Reply:
Because it is possible to use this field, without having to create a custom LSP notification. So we can get all additional capabilities information from the server as a result of the initialize request, thus we stay compatible with LSP specification.